Monitor your own workload - RNs 4.5 (#23270)
openshift/openshift-docs: release_notes/ocp-4-5-release-notes.adoc

{product-title} {product-version} introduces the Vertical Pod Autoscaler Operator (VPA). The VPA reviews the historic and current CPU and memory resources for containers in Pods and can update the resource limits and requests based on the usage values it learns. You create individual custom resources (CR) to instruct the VPA to update all of the Pods associated with a workload object, such as a Deployment, Deployment Config, StatefulSet, Job, DaemonSet, ReplicaSet, or ReplicationController. The VPA helps you to understand the optimal CPU and memory usage for your Pods and can automatically maintain Pod resources through the Pod lifecycle.

[id="ocp-4-5-cluster-monitoring"]
=== Cluster monitoring

> [id="ocp-4-5-monitor-your-own-workload-tp"]

Reviewer: ```suggestion
[id="ocp-4-5-monitor-your-own-services-tp"]
```

> ==== Monitor your own workload (Technology Preview)

Reviewer: ```suggestion
==== Monitor your own services (Technology Preview)
```

The following improvements are now available to further enhance monitoring your

> own cluster workload:

Reviewer: ```suggestion
own services:
```

> * Allow cross-correlation of in-cluster metrics.

Reviewer: ```suggestion
* Allow cross-correlation of the metrics of your own service with the cluster metrics.
```

> * Allow user workload monitoring metrics in user namespaces in recording and

Reviewer: ```suggestion
* Allow using metrics of services in user namespaces in recording and
```
